project/nsw-state-library-amplify/project.json

```diff
@@ -10,7 +10,8 @@
   "gaCode": "",
   "adminEmails": [
     "glen.crawford@reinteractive.net",
-    "riana.ferreira@reinteractive.net"
+    "riana.ferreira@reinteractive.net",
```

Comment:
@rtrvrtg , can we remove this email please -> riana.ferreira@reinteractive.net. Coz unfortunately `Riana` is no longer with reinteractive